src/tests/build.sh

```diff
@@ -541,6 +575,10 @@ __DistroRid=""
 __DoCrossgen2=0
 __CompositeBuildMode=0
 __TestBuildMode=
+__BuildTestProject=""
+__BuildTestDir=""
+__BuildTestTree=""
```

Comment:
nit: (like elsewhere in this script) empty string is not needed.

Reply:
Changed to "%3B" to overcome MSBuild parameter shenanigans.

Reply:
Heh, we also had to do similar thing once in https://github.com/dotnet/runtime/blob/03601a7a817068adb67639673db2010eaaf32d2a/eng/build.sh#L471-L473